packages/lib/src/session/SignIn.ts

```diff
@@ -53,17 +67,19 @@ export async function signIn(
         const account = (connection.account as Account).address;
 
         let deliveryServiceToken: string;
+        const nonce = 0;
+        const nonceMsg = getStorageKeyCreationMessage(nonce);
+        const signedNonceMsg = await personalSign(provider, account, nonceMsg);
 
-        if (!externalDataFile && !browserDataFile) {
-            const keys = await createKeys(
-                connection,
-                personalSign,
-                getSymmetricalKeyFromSignature,
-            );
+        const keys = await createKeys(
+            await createStorageKey(signedNonceMsg),
+            nonce,
+        );
 
+        if (!externalDataFile && !browserDataFile) {
```

Comment:
Is this just happen during development? Making a comment here would make sense to explain this code section.

Reply:
There will be an issue addressing the sing in / auth.